addons/l10n_pe/models/account_move.py

```diff
@@ -13,7 +13,7 @@ def _get_l10n_latam_documents_domain(self):
             return result
         if self.journal_id.type == "sale":
             result.append(("code", "in", ("01", "03", "07", "08", "20", "40")))
-        if self.partner_id.l10n_latam_identification_type_id.l10n_pe_vat_code != '6':
+        if self.journal_id.type == "sale" and self.partner_id.l10n_latam_identification_type_id.l10n_pe_vat_code != '6':
```

Comment:
```suggestion
        elif self.journal_id.type == "sale" and self.partner_id.l10n_latam_identification_type_id.l10n_pe_vat_code != '6':
```?

Reply:
or maybe in the previous if?

Reply:
Hello @jco-odoo 

What do you think about my last improve?

Regards